tests/ltp/Makefile

```diff
@@ -118,22 +119,27 @@ group-file:
 	echo "grp701:x:701:user700" >> appdir/etc/group
 	echo "grp704:x:704:" >> appdir/etc/group
 	echo "grp705:x:705:" >> appdir/etc/group
+	echo "grp1000:x:1000:" >> appdir/etc/group
 
 add_host_users_and_groups:
 	sudo groupadd -g 100701 grp701
 	sudo groupadd -g 100704 grp704
 	sudo groupadd -g 100705 grp705
+	sudo groupadd -g 101000 grp1000
```

Comment:
Sometimes the group might already be there from previous build. We should either append ` | ture` or check whether the group/user exists before the add commands.

Reply:
Fixed.